wire-ios/WireCommonComponents/FontScheme.swift

```diff
@@ -50,6 +51,11 @@ public enum FontWeight: String, CaseIterable {
     case black
 }
 
+public extension Font {
+    static var sfMonoSmall = Font.custom("SF Mono", size: 12)
```

Comment:
are we introducing new fonts for this screen?

Reply:
@netbe Based on the design, it is supposed to be SF Mono. I did not see that being used in the current code

Reply:
Yes. Based on the design, we have to use SF Mono. Currently, we use SF Pro Text.